src-shared/db-enum.js

```diff
@@ -135,7 +135,8 @@ exports.sessionOption = {
   manufacturerCodes: 'manufacturerCodes',
   profileCodes: 'profileCodes',
 }
-
+const clusterRevision = '48'
+exports.clusterRevision = clusterRevision
```

Comment:
Why do we need this?

Reply:
whoops, this was an experiment. deleting, thanks